.github/workflows/deploy-in-prod

```diff
@@ -0,0 +1,17 @@
+name: prod - update deployment in prod 
+
+on:
+  workflow_dispatch:
+
+jobs:
+  tag_for_prod:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Log into OpenShift
+        run: |
+          oc version
+          oc login --token=${{ secrets.OpenShiftToken }} --server=${{ secrets.OpenShiftServerURL }}
+      - name: Update imagestream prod
+        run: |      
+          oc -n de0974-dev tag mautic-subscription-description@dev mautic-subscription:prod
```

Comment:
Its called deploy in prod but we are running in dev namespace? also for tag we should use `:`

Reply:
and what is this namespace? I didn;t see anything there